consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/CliqueVotingBlockInterfaceTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.clique;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import tech.pegasys.pantheon.consensus.common.CastVote;
+import tech.pegasys.pantheon.consensus.common.ValidatorVotePolarity;
+import tech.pegasys.pantheon.crypto.SECP256K1.KeyPair;
+import tech.pegasys.pantheon.ethereum.core.Address;
+import tech.pegasys.pantheon.ethereum.core.AddressHelpers;
+import tech.pegasys.pantheon.ethereum.core.BlockHeader;
+import tech.pegasys.pantheon.ethereum.core.BlockHeaderBuilder;
+import tech.pegasys.pantheon.ethereum.core.BlockHeaderTestFixture;
+import tech.pegasys.pantheon.ethereum.core.Util;
+import tech.pegasys.pantheon.ethereum.mainnet.MainnetBlockHashFunction;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Test;
+
+public class CliqueVotingBlockInterfaceTest {
+
+  private final KeyPair proposerKeys = KeyPair.generate();
+  private final Address proposerAddress = Util.publicKeyToAddress(proposerKeys.getPublicKey());
+  private final List<Address> validatorList = singletonList(proposerAddress);
+
+  @Test
+  public void headerWithZeroCoinbaseReturnsAnEmptyVote() {
+    final BlockHeaderTestFixture headerBuilder = new BlockHeaderTestFixture();
+    headerBuilder.coinbase(AddressHelpers.ofValue(0));
+    final CliqueVotingBlockInterface serDeser = new CliqueVotingBlockInterface();
```

Comment:
We should just create this as a field in the class and avoid the weird abbreviation for a name.  Maybe just `blockInterface`?
`BlockHeaderTestFixture` can probably be created once as a class variable as well (JUnit creates a new instance of the test class for each method so class fields are not reused).

Reply:
done.